lib/fog/ovirt.rb

```diff
@@ -1,5 +1,6 @@
 require "fog/core"
 require "fog/xml"
+require "fog-json"
```

Comment:
the tests are failing currently: shouldn't this be 'fog/json'?

Reply:
yeah it should, it got messed up when resolving merge conflicts